Graphs/Bar/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+var BarChart = require('react-d3').BarChart;
```

Comment:
Rather than require the entire react-d3 library, find the module in the library that provides the `BarChart` and require only that.  Check out how the tux/Router works to see how to do this.